src/Voron/Data/Tables/Table.cs

```diff
@@ -524,7 +524,7 @@ internal Tree GetTree(Slice name)
         private Tree GetTree(TableSchema.SchemaIndexDef idx)
         {
             if (idx.IsGlobal)
-                return _tx.ReadTree(idx.Name);
+                return _tx.ReadTree(idx.NameAsSlice.ToString());
```

Comment:
Not that sure if considerable, but probably is not something you are required to do at a single time.